src/pages/index.js

```diff
@@ -129,28 +110,100 @@ function closeModal(modal) {
   modal.classList.remove("modal_opened");
   document.removeEventListener('keydown', handleEscKey);
   modal.removeEventListener('mousedown', handleOverlayClick);
+
+  const formEl = modal.querySelector("form");
+  if (formEl) {
+    resetForms(formEl);
+    resetValidation(formEl, settings);
```

Comment:
`resetValidation` is for clearing possible validation errors when you open the **profile** form, because you insert **valid** data into its inputs. 


It's not needed for all modals